### ISSUE TITLE
Quickstarter tests with sonar check fail due to unmatching golden

### ISSUE THREAD

**Issue by renedupont** [opendevstack/ods-quickstarters#431]
**Describe the bug**

When I run the quickstarter tests against our ods installation, the ones that do a sonar check fail because the quality gate key are different. 
The test expects
```
          "qualityGate": {
            "key": 1,
```
but I receive
```
          "qualityGate": {
            "key": "AXOA41Y0oYlOOizW5yiv",
```

Here is one example:

```
--- FAIL: TestJenkinsFile (456.01s)
    jenkinsfile_test.go:120: Actual sonar scan for run: docgen doesn't match -golden:
        '{
          "key": "unitt-docgen",
          "organization": "default-organization",
          "name": "unitt-docgen",
          "isFavorite": false,
          "visibility": "public",
          "extensions": [],
          "qualityProfiles": [
            {
              "name": "Sonar way",
              "language": "grvy",
              "deleted": false
            }
          ],
          "qualityGate": {
            "key": 1,
            "name": "Sonar way",
            "isDefault": true
          },
          "breadcrumbs": [
            {
              "key": "unitt-docgen",
              "name": "unitt-docgen",
              "qualifier": "TRK"
            }
          ]
        }
        '
        -sonar response:
        '{
          "key": "unitt-docgen",
          "organization": "default-organization",
          "name": "unitt-docgen",
          "isFavorite": false,
          "visibility": "public",
          "extensions": [],
          "qualityProfiles": [
            {
              "name": "Sonar way",
              "language": "grvy",
              "deleted": false
            }
          ],
          "qualityGate": {
            "key": "AXOA41Y0oYlOOizW5yiv",
            "name": "Sonar way",
            "isDefault": true
          },
          "breadcrumbs": [
            {
              "key": "unitt-docgen",
              "name": "unitt-docgen",
              "qualifier": "TRK"
            }
          ]
        }
        '
FAIL
FAIL	github.com/opendevstack/ods-quickstarters/tests/ods-document-gen-svc	456.594s
```

fyi @georgfedermann @clemensutschig

**Comment by clemensutschig** [comment]
Xak ..  this a Quick fix 
https://github.com/opendevstack/ods-quickstarters/blob/master/tests/scripts/print-sonar-scan-run.sh#L3 add a del to the end and then remove the key from the golden Files as well

**Comment by renedupont** [comment]
was fixed in #430
